src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(callback, operand) {
+      callback(operand);
+
+      return calculator;
+    },
+    add(num) {
+      calculator.result += num;
```

Comment:
better to use `this` instead of the calculator, it will improve the reusability
```suggestion
    add(num) {
      this.result += num;
```